financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerViewModel.kt

```diff
@@ -83,12 +81,7 @@ internal class AccountPickerViewModel @Inject constructor(
                     activeAuthSession.skipAccountSelection == true,
                 accounts = accounts,
                 selectionMode = if (manifest.singleAccount) SelectionMode.RADIO else SelectionMode.CHECKBOXES,
-                accessibleData = AccessibleDataCalloutModel(
-                    businessName = ConsentTextBuilder.getBusinessName(manifest),
-                    permissions = manifest.permissions,
-                    isStripeDirect = manifest.isStripeDirect ?: false,
-                    dataPolicyUrl = FinancialConnectionsUrlResolver.getDataPolicyUrl(manifest)
-                ),
+                accessibleData = AccessibleDataCalloutModel.fromManifest(manifest),
```

Comment:
adding a note on whether we filter permissions somewhere

https://github.com/stripe/stripe-ios/blob/2a8dbdc3/StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/MerchantDataAccessView.swift#L49-L54